http2_general/test_h2_headers.py

```diff
@@ -1407,3 +1407,74 @@ def test_pragma_from_hpack_table(self):
         self.assertEqual(3, len(server.requests))
         client.send_request(request, "200")
         self.assertEqual(3, len(server.requests))
+
+
+class TestHeadersBlockedByMaxHeaderListSize(tester.TempestaTest):
```

Comment:
Maybe you should add tests with headers equal to 200 bytes and less than 200 bytes?

Reply:
Yes fixed add new tests